src/types/SeisCon.jl

```diff
@@ -20,6 +20,13 @@ function getindex(con::SeisCon, a::Colon)
     read_con(con, 1:length(con))
 end
 
+"""
+    copy(s::SeisCon)
+
+Create a copy of SeisCon object.
+"""
+copy(s::SeisCon) = SeisCon(s.ns, s.dsf, s.blocks)
```

Comment:
copy is a Base function which is why you cmget that Warning. You need to import it then extend it and not export it 

```
Import Base.copy

copy(.....
```

You also need to call copy on the blocks or the new object gonna refer to the same block unsafely

Reply:
right, thank you!